src/cli/telemetry/telemetry.ts

```diff
@@ -143,27 +144,29 @@ export const prepareData = async (
   const build = coreCompiler.buildId || 'unknown';
   const has_app_pwa_config = hasAppTarget(config);
   const anonymizedConfig = anonymizeConfigForTelemetry(config);
+  const isBrowserEnv = IS_BROWSER_ENV;
 
   return {
-    yarn,
-    duration_ms,
+    arguments: config.flags.args,
+    build,
     component_count,
-    targets,
+    config: anonymizedConfig,
+    cpu_model,
+    duration_ms,
+    has_app_pwa_config,
+    isBrowserEnv,
+    os_name,
+    os_version,
     packages,
     packages_no_versions: packagesNoVersions,
-    arguments: config.flags.args,
-    task: config.flags.task,
+    rollup,
     stencil,
     system,
     system_major: getMajorVersion(system),
-    os_name,
-    os_version,
-    cpu_model,
-    build,
+    targets,
+    task: config.flags.task,
     typescript,
-    rollup,
-    has_app_pwa_config,
-    config: anonymizedConfig,
+    yarn,
```

Comment:
most of the diff here is just from alphabetizing these values while I was in here